messages/src/main/java/life/genny/messages/process/MessageProcessor.java

```diff
@@ -89,7 +89,7 @@ private void processMessageHelper(QMessageGennyMSG message) {
 
         log.debug("Realm is " + realm + " - Incoming Message :: " + message.toString());
 
-        BaseEntity projectBe = beUtils.getBaseEntity("PRJ_" + realm.toUpperCase());
+        BaseEntity projectBe = beUtils.getBaseEntity("PRJ_" + realm.toUpperCase(), true);
```

Comment:
Avoid bundling the attributes. Instead, use EntityAttributeUtils to fetch individual values.